### DISCUSSION TITLE
AutoFill in compression menu(iOS)

### DISCUSSION THREAD

**Discussion by hiverdoux** [aonez/Keka#1370] [Ideas]
Cannot use AutoFill in password field when compressing, whereas I can when extracting archive. Please consider implementing AutoFill in compress menu too. Thanks.

**Comment by aonez** [comment]
Thanks for the tip @hiverdoux. Can you send some screenshots of the issue? Also what Keka and iOS version are you using?

**Reply by hiverdoux** [reply]
![IMG_2668](https://github.com/aonez/Keka/assets/135621924/d64d54ae-7cfa-44f9-95a4-95657e9fa17a)
This is the autofill popup I meant. (And iOS automatically hides password autofill popup and keyboard image in screenshot)
I'm using both latest version of Keka and iOS.
Also sorry for the delayed reply. Thanks.
